From: davidm@hpl.hp.com (David Mosberger)
Subject: Re: problem compiling 2.4.9
Newsgroups: list.linux.ia64
Date: 21 Aug 2001 09:44:15 -0600
Organization: GAG News/Mail Gateway
Reply-To: davidm@hpl.hp.com

The attached patch accidentally was dropped from the 2.4.9 patch I put
out yesterday.  I have updated the patch at linux.kernel.org but if
you already got it, just apply the patch below and you should be fine.
Sorry about the inconvenience.

	--david

diff -urN linux-2.4.9/include/asm-ia64/keyboard.h linux-2.4.9-lia/include/asm-ia64/keyboard.h
--- linux-2.4.9/include/asm-ia64/keyboard.h	Fri Apr 21 16:38:40 2000
+++ linux-2.4.9-lia/include/asm-ia64/keyboard.h	Tue Aug 21 08:31:41 2001
@@ -2,16 +2,16 @@
 #define _ASM_IA64_KEYBOARD_H
 
 /*
- * This file contains the ia-64 architecture specific keyboard
- * definitions.
+ * This file contains the ia64 architecture specific keyboard definitions.
  *
- * Copyright (C) 1998, 1999 Hewlett-Packard Co
- * Copyright (C) 1998, 1999 David Mosberger-Tang <davidm@hpl.hp.com>
+ * Copyright (C) 1998, 1999, 2001 Hewlett-Packard Co
+ * Copyright (C) 1998, 1999, 2001 David Mosberger-Tang <davidm@hpl.hp.com>
  */
 
 # ifdef __KERNEL__
 
 #include <linux/irq.h>
+#include <linux/kd.h>
 
 #define KEYBOARD_IRQ			isa_irq_to_vector(1)
 #define DISABLE_KBD_DURING_INTERRUPTS	0
@@ -23,6 +23,7 @@
 			   char raw_mode);
 extern char pckbd_unexpected_up(unsigned char keycode);
 extern void pckbd_leds(unsigned char leds);
+extern int pckbd_rate(struct kbd_repeat *rep);
 extern void pckbd_init_hw(void);
 extern unsigned char pckbd_sysrq_xlate[128];
 
@@ -32,12 +33,14 @@
 #define kbd_translate		pckbd_translate
 #define kbd_unexpected_up	pckbd_unexpected_up
 #define kbd_leds		pckbd_leds
+#define kbd_rate		pckbd_rate
 #define kbd_init_hw		pckbd_init_hw
 #define kbd_sysrq_xlate		pckbd_sysrq_xlate
 
 #define INIT_KBD
 
 #define SYSRQ_KEY		0x54
+#define E1_PAUSE   119		/* PAUSE key */
 
 /* resource allocation */
 #define kbd_request_region()

